src/chrome/content/rules/Rephial.org.xml

```diff
@@ -0,0 +1,14 @@
+<!--
+    Redirect to HTTP:	
+    buildbot.rephial.org
+
+-->
+<ruleset name="Rephial.org">
+	<target host="rephial.org" />
+	<target host="www.rephial.org" />
```

Comment:
@RemakingEden Thank you for your contribution! I'm glad HTTPSE could be your first open source project. Glad to have you.

It looks like those TLS certs are only for `trac.rephial.org` and `rephial.org`, but not `www.trac.rephial.org` and `www.rephial.org`. Many web devevlopers miss adding both in for proper HTTPS configuration coverage for their domain

1. You can take several actions. To fix the test, you can add comments above in the that these `www` hosts are nonfunctional
2. Contact the website admin to amend their TLS Cert
They seem to use Let's Encrypt, so assuming they probably used Certbot to install...they can use this link for reference
```suggestion
```
https://certbot.eff.org/docs/using.html#changing-a-certificate-s-domains